### ISSUE TITLE
Cannot add account ("Error adding your athenticator")

### ISSUE THREAD

**Issue by VirtualNick** [Jessecar96/SteamDesktopAuthenticator#1017]
I click "Setup New Account", enter my details and continue.
After that it asks me to enter the code that I got on my e-mail. No problems so far. I get another e-mail where it asks me if I want to add a mobile phone number to my steam account. I click that link and continue on the desktop authenticator. 

Then the program asks me to put in my phone number. I continue and all I get is  "Error adding your athenticator". The thing is that I actually receive a code from steam via SMS. 

Now I cannot even get to the step where I can enter my phone number anymore. The last thing I see is "Steam account login succeeded. Press OK to continue adding SDA as your authenticator." after putting in the code from the first e-mail you get in the process. When I click "OK" on that message, I get the same error as above.

The funny thing is:
I recently got scammed by using on of the fake authenticators. The fake authenticator showed that it used the version 1.0.14 instead of the legit 1.0.13. Thats why I tried it. And IT WORKED WITH THE FAKE AUTHENTICATOR but not with the original one. How is that possible? I dont get it.

**Comment by Rickard03** [comment]
Having same issue but getting "Object refernce not set to an instance"

**Comment by excellproj** [comment]
Some problem
SDA 1.0.13

**Comment by realBernieReid** [comment]
Jesse! Come on & update the app!!! We're counting on you, so we can get back to business! Too many scam artists are forking your model with their hacking software programs. What's taking so long to get a response from you?

**Comment by Xanstein** [comment]
Same bug as everyone else, please make an update)

**Comment by VirtualNick** [comment]
2 weeks and not even an answer from him here...

Let's be honest, he will not update it. I would suggest you to use the official steam app as authenticator. You can also set up multiple accounts there. For many people that might not be an option, but for most that should work.

The malicious version of sda that comes with a virus still works btw (tested with a throwaway account). Is he even working with the scammers or encouraging it? Any form of communication would be nice, because of the scams happening... I don't trust this program anymore.

**Comment by realBernieReid** [comment]
> 2 weeks and not even an answer from him here...
> 
> Let's be honest, he will not update it. I would suggest you to use the official steam app as authenticator. You can also set up multiple accounts there. For many people that might not be an option, but for most that should work.
> 
> The malicious version of sda that comes with a virus still works btw (tested with a throwaway account). Is he even working with the scammers or encouraging it? Any form of communication would be nice, because of the scams happening... I don't trust this program anymore.

Go to his discord and spam them! They work on the Marketplace.tf discord.

**Comment by ZangYUzhang** [comment]
The issue of requesting an update！ cant add a new Steam Authenticator on SDA

**Comment by globart** [comment]
> Having same issue but getting "Object refernce not set to an instance"

To fix this, you have to remove Steam Guard completely and then do the setup in the SDA

**Comment by globart** [comment]
> I click "Setup New Account", enter my details and continue. After that it asks me to enter the code that I got on my e-mail. No problems so far. I get another e-mail where it asks me if I want to add a mobile phone number to my steam account. I click that link and continue on the desktop authenticator.
> 
> Then the program asks me to put in my phone number. I continue and all I get is "Error adding your athenticator". The thing is that I actually receive a code from steam via SMS.
> 
> Now I cannot even get to the step where I can enter my phone number anymore. The last thing I see is "Steam account login succeeded. Press OK to continue adding SDA as your authenticator." after putting in the code from the first e-mail you get in the process. When I click "OK" on that message, I get the same error as above.
> 
> The funny thing is: I recently got scammed by using on of the fake authenticators. The fake authenticator showed that it used the version 1.0.14 instead of the legit 1.0.13. Thats why I tried it. And IT WORKED WITH THE FAKE AUTHENTICATOR but not with the original one. How is that possible? I dont get it.

There is version 1.0.14, which is in "pre-release" status for some reason, but it works, just tested. You can just unpack the files to your SDA folder with option to replace existing ones. Here is the link - https://github.com/Jessecar96/SteamDesktopAuthenticator/releases/tag/1.0.14

**Comment by Armin2208** [comment]
> There is version 1.0.14, which is in "pre-release" status for some reason, but it works, just tested. You can just unpack the files to your SDA folder with option to replace existing ones. Here is the link - https://github.com/Jessecar96/SteamDesktopAuthenticator/releases/tag/1.0.14

just wanted to say thank you! you saved me.

**Comment by realBernieReid** [comment]
> > > I click "Setup New Account", enter my details and continue. After that it asks me to enter the code that I got on my e-mail. No problems so far. I get another e-mail where it asks me if I want to add a mobile phone number to my steam account. I click that link and continue on the desktop authenticator.
> > > Then the program asks me to put in my phone number. I continue and all I get is "Error adding your athenticator". The thing is that I actually receive a code from steam via SMS.
> > > Now I cannot even get to the step where I can enter my phone number anymore. The last thing I see is "Steam account login succeeded. Press OK to continue adding SDA as your authenticator." after putting in the code from the first e-mail you get in the process. When I click "OK" on that message, I get the same error as above.
> > > The funny thing is: I recently got scammed by using on of the fake authenticators. The fake authenticator showed that it used the version 1.0.14 instead of the legit 1.0.13. Thats why I tried it. And IT WORKED WITH THE FAKE AUTHENTICATOR but not with the original one. How is that possible? I dont get it.
> > 
> > 
> > There is version 1.0.14, which is in "pre-release" status for some reason, but it works, just tested. You can just unpack the files to your SDA folder with option to replace existing ones. Here is the link - https://github.com/Jessecar96/SteamDesktopAuthenticator/releases/tag/1.0.14
> 
> Thank you

Yeah, but the issue is that it's in Pre-Release testing. So there might be some major flaws. I won't be touching it until Jesse finally finishes it and puts it in the release section. But I'm getting tired of waiting and might just grab a burner phone to get this over with.

**Comment by vrmattos** [comment]
> > > > I click "Setup New Account", enter my details and continue. After that it asks me to enter the code that I got on my e-mail. No problems so far. I get another e-mail where it asks me if I want to add a mobile phone number to my steam account. I click that link and continue on the desktop authenticator.
> > > > Then the program asks me to put in my phone number. I continue and all I get is "Error adding your athenticator". The thing is that I actually receive a code from steam via SMS.
> > > > Now I cannot even get to the step where I can enter my phone number anymore. The last thing I see is "Steam account login succeeded. Press OK to continue adding SDA as your authenticator." after putting in the code from the first e-mail you get in the process. When I click "OK" on that message, I get the same error as above.
> > > > The funny thing is: I recently got scammed by using on of the fake authenticators. The fake authenticator showed that it used the version 1.0.14 instead of the legit 1.0.13. Thats why I tried it. And IT WORKED WITH THE FAKE AUTHENTICATOR but not with the original one. How is that possible? I dont get it.
> > > 
> > > 
> > > There is version 1.0.14, which is in "pre-release" status for some reason, but it works, just tested. You can just unpack the files to your SDA folder with option to replace existing ones. Here is the link - https://github.com/Jessecar96/SteamDesktopAuthenticator/releases/tag/1.0.14
> > 
> > 
> > Thank you
> 
> Yeah, but the issue is that it's in Pre-Release testing. So there might be some major flaws. I won't be touching it until Jesse finally finishes it and puts it in the release section. But I'm getting tired of waiting and might just grab a burner phone to get this over with.

friend, just download 1.14 create add account.

get the mafile files from your account and import them into version 1.13

My files are from the 2016 versions and I use them safely in 1.0.13

**Comment by realBernieReid** [comment]
> > > > > I click "Setup New Account", enter my details and continue. After that it asks me to enter the code that I got on my e-mail. No problems so far. I get another e-mail where it asks me if I want to add a mobile phone number to my steam account. I click that link and continue on the desktop authenticator.
> > > > > Then the program asks me to put in my phone number. I continue and all I get is "Error adding your athenticator". The thing is that I actually receive a code from steam via SMS.
> > > > > Now I cannot even get to the step where I can enter my phone number anymore. The last thing I see is "Steam account login succeeded. Press OK to continue adding SDA as your authenticator." after putting in the code from the first e-mail you get in the process. When I click "OK" on that message, I get the same error as above.
> > > > > The funny thing is: I recently got scammed by using on of the fake authenticators. The fake authenticator showed that it used the version 1.0.14 instead of the legit 1.0.13. Thats why I tried it. And IT WORKED WITH THE FAKE AUTHENTICATOR but not with the original one. How is that possible? I dont get it.
> > > > 
> > > > 
> > > > There is version 1.0.14, which is in "pre-release" status for some reason, but it works, just tested. You can just unpack the files to your SDA folder with option to replace existing ones. Here is the link - https://github.com/Jessecar96/SteamDesktopAuthenticator/releases/tag/1.0.14
> > > 
> > > 
> > > Thank you
> > 
> > 
> > Yeah, but the issue is that it's in Pre-Release testing. So there might be some major flaws. I won't be touching it until Jesse finally finishes it and puts it in the release section. But I'm getting tired of waiting and might just grab a burner phone to get this over with.
> 
> friend, just download 1.14 create add account.
> 
> get the mafile files from your account and import them into version 1.13
> 
> My files are from the 2016 versions and I use them safely in 1.0.13

I prefer waiting until 1.14 is fully released. Unreleased software updates can be a bit unpredictable, and I like to steer clear until everything is polished and finalized!

**Comment by Armin2208** [comment]
Oh new stable release came out today and it is the last update. Sad! But thanks for the last update dev.
